src/publish.test.ts

```diff
@@ -91,5 +91,38 @@ describe('publish', function () {
         }
       `);
     });
+
+    it('warns that a version exists if we are trying to release', async function () {
```

Comment:
nice test!!!

Reply:
it's the only part of the code that uses the library I wanted to update so I needed to add it 😂